scripts/AudioManager.js

```diff
@@ -83,16 +83,41 @@ function AudioManager(addEventListener, isTownTune) {
 					audio.volume -= step;
 				} else {
 					clearInterval(fade);
-					audio.pause();
+					playPause(false);
 					audio.volume = oldVolume;
 					if (callback) callback();
 				}
 			}, 100);
 		}
 	}
+	
+	function playPause(play) {
```

Comment:
I'm confused why this is a separate function instead of using audio.play and audio.pause.